packages/@rnw-scripts/create-github-releases/src/createGithubReleases.ts

```diff
@@ -215,6 +237,46 @@ function createReleaseMarkdown(release: Release): string {
   return md;
 }
 
+/**
+ * Finds a link to the the first stable/prerelease release notes of the major
+ * version. These notes are curated, and we want them to be visible on all
+ * releases.
+ */
+function getFirstMajorVersionLink(
+  release: Release,
+  allReleases: Release[],
+): string | null {
+  const releaseSemver = semver.parse(release.version);
+  if (!releaseSemver) {
+    return null;
+  }
+
+  let firstVersion: semver.SemVer;
+  if (releaseSemver.prerelease.length === 0) {
+    firstVersion = semver.parse(
+      `${releaseSemver.major}.${releaseSemver.minor}.0`,
+    )!;
+  } else if (
+    releaseSemver.prerelease.length === 2 &&
+    releaseSemver.prerelease[0] === 'preview'
+  ) {
+    firstVersion = semver.parse(
+      `${releaseSemver.major}.${releaseSemver.minor}.${releaseSemver.patch}.${releaseSemver.prerelease[0]}.1`,
```

Comment:
Why the `.1` at the end?

Reply:
It looks like we always start the preview at .1, not .0.

Reply:
Yep. This differs from upstream rc.0, and is something of an artifact of bumping with beachball. We always bump before publish, so we bump 0 to 1.

We start out with a ".0" patch release because we are bumping from a prerelease to instead go to a patch version.